### DISCUSSION TITLE
bootstrap toast UI

### DISCUSSION THREAD

**Discussion by SmashPlusUltra** [fkhadra/react-toastify#571] [Q&A] [answered]
Is there a way to get the same style like,..having header - body style.?
![image](https://user-images.githubusercontent.com/53838441/108279481-6d9e8500-715b-11eb-9f5a-8b895e64c3ef.png)

**Comment by fkhadra** [comment, marked as answer]
Hey @SmashPlusUltra, I've reproduced the bootstrap style below
[![Edit toastify-bootstrap](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/toastify-bootstrap-o0dv7?fontsize=14&hidenavigation=1&theme=dark)

I've left some comments to explain how it's done. Start with the `style.css` file. Hope this help

**Reply by SmashPlusUltra** [reply]
I really appreciate your time an effort to answer, that's great. Thanks!

**Reply by fkhadra** [reply]
You're welcome 👌.
